site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+description: >
+  Driving changes for better loading of third-party resources on the web
+layout:
+  'layouts/blog-post.njk'
+date: 2022-08-07
+authors:
+  - addyosmani
+hero: 'image/1L2RBhCLSnXjCnSlevaDjy3vba73/kng4NSnIkUQ9BU4RLT3p.jpeg'
+alt: >
+  ALT TEXT
+tags:
+  - performance
+  - news
+---
+
+[Third-party](https://almanac.httparchive.org/en/2021/third-parties) (3P) scripts and embeds are heavily used across the web today. They provide out-of-the-box solutions for embedding social media, videos, analytics, live chat, advertising, A/B testing, personalization, and others. Third-party embeds are a necessary part of modern websites that allow site owners to focus on their core competencies, while offloading standard but critical functions to external providers.
+
+When both first party and third parties on a web page work in harmony, we have a very performant outcome and a great user experience. However, this requires significant effort from both engineering and business teams and is often overlooked, resulting in less than performant web pages and a [negative impact](https://web.dev/third-party-javascript/#performance) to user-centric metrics such as the [Core Web Vitals](https://web.dev/vitals/). This is detrimental to both parties and creates missed opportunities in businesses. Could we do better here?
+
+We have a future vision of the web where third-party scripts and resources provide the intended business value with minimal regression to the performance or user experience of websites that use them in the browser. This would allow users to experience, ideally, faster page loads.
+
+Over the last year, we have considered, discussed, and experimented with many possibilities that can potentially protect web user experience from the detrimental impact of third-party scripts without reducing their business value to site-owners.
+
+Through this post, we wish to share information about some of our experiments. We hope this is the beginning of a process that will encourage transparency and visibility between user agents, businesses, and third-party providers and pave the path to a faster web.
+
+## A deeper look at third-parties
+
+A third-party is a resource served by a provider external to the site. They are not directly within the site owners' control, but present with their approval. Third-party resources are:
+
+-   Hosted on a shared and public origin different from the primary site origin.
+-   Not authored or influenced by an individual site owner.
+-   Widely used by a variety of sites.
+
+From helping to generate revenue (via ads) to providing better marketing opportunities (social media embeds), third-parties serve a wide variety of business objectives. Common categories of third-parties include the following:
+
+Source: [Third-parties by category](https://github.com/patrickhulce/third-party-web/#third-parties-by-category).
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
```

Comment:
Are these `<br>`s strictly necessary for formatting? If not, I'd drop them so the table cells can display as they should by default.